### PR TITLE
[new release] iostream (2 packages) (0.2.1)

### PR DESCRIPTION
**Pull request by c-cube**
Generic, composable IO input and output streams

- Project page: <a href="https://github.com/c-cube/ocaml-iostream">https://github.com/c-cube/ocaml-iostream</a>
- Documentation: <a href="https://c-cube.github.io/ocaml-iostream">https://c-cube.github.io/ocaml-iostream</a>

##### CHANGES:

- bugfix for iostream-camlzip

### CODE REVIEW

**Comment by c-cube** [comment]
it's just the usual blurb about `dune subst`, which is not used anyway.

File: packages/iostream-camlzip/iostream-camlzip.0.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+bug-reports: "https://github.com/c-cube/ocaml-iostream/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.0"}
+  "iostream" {= version}
+  "camlzip"
+  "ounit2" {with-test}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/iostream-camlzip/iostream-camlzip.0.2.1/opam:18]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/iostream/iostream.0.2.1/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+doc: "https://c-cube.github.io/ocaml-iostream"
+bug-reports: "https://github.com/c-cube/ocaml-iostream/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.0"}
+  "ounit2" {with-test}
+]
+depopts: ["base-unix"]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/iostream/iostream.0.2.1/opam:17]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
Thanks

**Comment by c-cube** [comment]
thank you!!

**Pull request merged**